src/io/flutter/editor/NativeEditorNotificationProvider.java

```diff
@@ -77,6 +77,9 @@ else if (root.getName().equals("ios")) {
   }
 
   private static VirtualFile findRootDir(@NotNull VirtualFile file, VirtualFile projectDir) {
+    if (projectDir == null) {
```

Comment:
👍 

Perhaps also mark the param as `@Nullable` (and the method return as well)?

Reply:
Good call.  Done.